Plugins/Power Manager/Devices & Monitoring List/Python/get_power_manager_capable_devices.py

```diff
@@ -0,0 +1,221 @@
+#
+#  Python script using OpenManage Enterprise API to get device list which are capable to be monitored/managed by Power Manager.
+#
+# _author_ = Mahendran P <Mahendran_P@Dell.com>
+# _version_ = 0.2
+#
+#
+# Copyright (c) 2020 Dell EMC Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""
+SYNOPSIS:
+   Script to get the list of devices from OpenManage Enterprise which are capable to be monitored/managed by Power Manager
+
+DESCRIPTION:
+   This script exercises the OpenManage Enterprise REST API to get a list of devices currently being managed by OpenManage Enterprise & capable to be monitored/managed by Power Manager.
+    - For authentication, X-Auth is used over Basic Authentication
+    - Note that the credentials entered are not stored to disk.
+
+EXAMPLE:
+   python Get_Power_Manager_Capable_Devices.py --ip <xx> --username <username> --password <pwd>
+   
+    Output:
+    
+       =====================================================
+          Devices List with Power Manager capablilities
+       =====================================================
+
+      DEVICE_ID  SERVICE_TAG  MODEL            DEVICE_NAME              POWER_MANAGER_CAPABILITY
+
+      10113      XXXXXXX      PowerEdge R640   WIN2K12356.BLR.net       Monitor + Management
+      10106      XXXXXXX      PowerEdge R640   WINDOWS2019.BLR.net      Monitor only
+      10105      XXXXXXX      PowerEdge R640   WINHIRTK12.BLR.net       Monitor + Management
+      10111      XXXXXXX      PowerEdge R640   WINKKLLLL.BLR.net        Monitor only
+      10109      XXXXXXX      PowerEdge R640   WIN2019SCALAB.BLR.net    Monitor + Management
+
+"""
+
+#Import the modules required for this script
+import sys
+import argparse
+from argparse import RawTextHelpFormatter
+import json
+import requests
+import urllib3
+from columnar import columnar
```

Comment:
OME core has been trying to eliminate the use of additional modules that need to be user installed - would be nice to get rid of these additional columns and default to csv / json type output formats - for powershell you could leverage existing constructs to print in a table format